src/firebase/firebase.ts

```diff
@@ -10,4 +12,4 @@ export const auth = getAuth(app);
 export const db = getFirestore(app);
 export const gitHubProvider = new GithubAuthProvider();
 export const googleProvider = new GoogleAuthProvider();
-export const usersCollection = collection(db, 'users');
+export const usersCollection = collection(db, NamesDBCollection.Users);
```

Comment:
There should not be such a constant since we should be able to work with other collections. See comment above